lxd/storage_volumes_backup.go

```diff
@@ -191,31 +190,29 @@ func storagePoolVolumeTypeCustomBackupsGet(d *Daemon, r *http.Request) response.
 		return response.BadRequest(fmt.Errorf("Invalid storage volume type %q", volumeTypeName))
 	}
 
-	var poolID int64
-
-	err = s.DB.Cluster.Transaction(r.Context(), func(ctx context.Context, tx *db.ClusterTx) error {
-		var err error
-
-		poolID, _, _, err = tx.GetStoragePool(ctx, poolName)
-
-		return err
-	})
+	pool, err := request.GetCtxValue[storagePools.Pool](r.Context(), ctxStoragePool)
 	if err != nil {
 		return response.SmartError(err)
 	}
 
+	// FIXME: Is handler supposed to forward requests if the target is remote?
```

Comment:
I was confused here as to why this handler forwards requests to other cluster members if the volume is remote but not when the target parameter is remote. Added the FIXME here so we can discuss before merging.

Reply:
I've added an issue to track this here: https://github.com/canonical/lxd/issues/13548